libs/common/to_lower.hpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TO_LOWER_HPP
+#define IROHA_TO_LOWER_HPP
+
+#include <string>
+
+namespace iroha {
+
+  inline std::string &toLower(std::string_view src, std::string &dst) {
```

Comment:
You are always appending. As this is a library function, somebody could think it replaces dst. I suggest reset dst before using.

```
dst = {};
```

If not, library function name should be obvious `toLowerAppend()`.

Reply:
```dst``` can be cached and contain some data before call.

Reply:
```toLowerAppend``` agree